### PR TITLE
[10.0][MIG] Extract/New module account_fiscal_position_rule_sale_stock

### PR DESCRIPTION
**Pull request by mbcosta**
Extracted code refers to Sale/Procurement from module account_fiscal_position_rule_stock to this new one to have a better modularization, suggested by @renatonlima in review of account_fiscal_position_rule_stock https://github.com/OCA/account-fiscal-rule/pull/90

cc @rvalyi

### CODE REVIEW

File: account_fiscal_position_rule_sale_stock/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+No configuration required.
+
+Changelog
+=========
+
+10.0.1.0.0 (2019-08-16)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Created module, it's a extraction of account_fiscal_rule_stock to better modularization.
```
Comment (rvalyi) [inline account_fiscal_position_rule_sale_stock/README.rst]: s/Module creation./It is an extraction from account_fiscal_rule_stock for a better modularization./

File: account_fiscal_position_rule_sale_stock/readme/HISTORY.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+10.0.1.0.0 (2019-08-16)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Module creation.It is an extraction from account_fiscal_rule_stock for a better modularization.
```
Comment (rvalyi) [inline account_fiscal_position_rule_sale_stock/readme/HISTORY.rst]: Missing space:
s/Module creation.It/Module creation. It/

File: account_fiscal_position_rule_sale_stock/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This module extends the sale stock.
```
Comment (rvalyi) [inline account_fiscal_position_rule_sale_stock/readme/DESCRIPTION.rst]: I suggest instead:

This module enables the account_fiscal_position_rule features (see module README) in the context of a sale order with a delivery order (sale and stock modules).

**Review by rvalyi — CHANGES_REQUESTED** [review]
Thank you for the module extraction @mbcosta. Minor changes in the README requested though.

**Comment by mbcosta** [comment]
Hi @rvalyi thanks for review, I made the changes

**Review by rvalyi — APPROVED** [review]
ok for me. thanks!

**Review by renatonlima — APPROVED** [review]

**Review by mileo — APPROVED** [review]

**Comment by mbcosta** [comment]
Hi @renatonlima @mileo , thanks by review

Hi @rvalyi , it's seems the command doesn't work because Travis are freeze https://travis-ci.org/OCA/account-fiscal-rule/builds/584639961?utm_source=github_status&utm_medium=notification

**Pull request merged**